src/components/ebay-carousel/README.md

```diff
@@ -23,6 +23,7 @@ Name | Type | Stateful | Required | Description
 `index` | String | Yes | No | 0-based index position
 `items-per-slide` | String | No | No | automatically fit a number of items for each carousel slide and enable slide controls. If set to a whole number, will default to x.1 where x is the whole number set.
 `gap` | String | No | No | override the margin between carousel items in pixels (default: "16")
+`image-treatment` | Boolean | No | No | If true, image treatment styles are applied. Default is false.
```

Comment:
I think an enumerated type would give more room to grow for other potential image treatments. So, `image-treatment=none` by default, then maybe `image-treatment=matte`by default?

Reply:
If we like the term `matte`, then we could also maybe use it in our skin classname...?

Reply:
Yeah we could do that... but I mean... YAGNI? 😁

Reply:
Oh, I don't know. Maybe an `image-treatment=warhol` easter egg? Or `image-treatment=cubism`... if we get all artsy and highbrow 😁

Reply:
Haha sorry about having to google YAGNI. 

I updated image-treatment to be an enum 👍